features/admin.applications.v1/pages/applications.tsx

```diff
@@ -596,8 +599,11 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
         >
             {
                 (
-                    !isMyAccountApplicationDataFetchRequestLoading
-                    && myAccountApplicationData?.applications?.length !== 0
+                    legacyAuthzRuntime
```

Comment:
Let's add a comment here explaining the logic, to prevent such cases from happening again 😁

Reply:
Let's add a comment in a follow up PR